<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SplashScreen</class>
 <widget class="QMainWindow" name="SplashScreen">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>680</width>
    <height>400</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <property name="spacing">
     <number>0</number>
    </property>
    <property name="leftMargin">
     <number>10</number>
    </property>
    <property name="topMargin">
     <number>10</number>
    </property>
    <property name="rightMargin">
     <number>10</number>
    </property>
    <property name="bottomMargin">
     <number>10</number>
    </property>
    <item>
     <widget class="QFrame" name="dropShadowFrame">
      <property name="styleSheet">
       <string notr="true">QFrame {	
	background-color: rgb(56, 58, 89);	
	color: rgb(220, 220, 220);
	border-radius: 10px;
}</string>
      </property>
      <property name="frameShape">
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow">
       <enum>QFrame::Raised</enum>
      </property>
      <widget class="QLabel" name="label_title">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>90</y>
         <width>661</width>
         <height>61</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>40</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(254, 121, 199);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;YOUR&lt;/strong&gt; APP NAME</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_description">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>150</y>
         <width>661</width>
         <height>31</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 114, 164);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;APP&lt;/strong&gt; DESCRIPTION</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QProgressBar" name="progressBar">
       <property name="geometry">
        <rect>
         <x>50</x>
         <y>280</y>
         <width>561</width>
         <height>23</height>
        </rect>
       </property>
       <property name="styleSheet">
        <string notr="true">QProgressBar {
	
	background-color: rgb(98, 114, 164);
	color: rgb(200, 200, 200);
	border-style: none;
	border-radius: 10px;
	text-align: center;
}
QProgressBar::chunk{
	border-radius: 10px;
	background-color: qlineargradient(spread:pad, x1:0, y1:0.511364, x2:1, y2:0.523, stop:0 rgba(254, 121, 199, 255), stop:1 rgba(170, 85, 255, 255));
}</string>
       </property>
       <property name="value">
        <number>24</number>
       </property>
      </widget>
      <widget class="QLabel" name="label_loading">
       <property name="geometry">
        <rect>
         <x>0</x>
         <y>320</y>
         <width>661</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>12</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 114, 164);</string>
       </property>
       <property name="text">
        <string>loading...</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
      <widget class="QLabel" name="label_credits">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>350</y>
         <width>621</width>
         <height>21</height>
        </rect>
       </property>
       <property name="font">
        <font>
         <family>Segoe UI</family>
         <pointsize>10</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true">color: rgb(98, 114, 164);</string>
       </property>
       <property name="text">
        <string>&lt;strong&gt;Created&lt;/strong&gt;: Wanderson M. Pimenta</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
